doc/Type/Backtrace.pod6

```diff
@@ -77,16 +77,16 @@ Defined as:
 Returns a summary string representation of the backtrace, filtered
 by C<!.is-hidden && (.is-routine || !.is-setting)>.
 
+    this program:
     sub inner { say Backtrace.new.summary }
     sub outer { inner; }
     outer;
 
-    #OUTPUT:
-    #perl6 test.p6
-    #  in method new at SETTING::src/core/Backtrace.pm6 line 85
-    #  in sub inner at test.p6 line 1
-    #  in sub outer at test.p6 line 2
-    #  in block <unit> at test.p6 line 3
+    results in:
```

Comment:
And this? Also try and run changed files through examples-compilation.t to detect this